homeassistant/components/hive/strings.json

```diff
@@ -0,0 +1,46 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Hive Login",
+        "description": "Enter your Hive login information and configuration.",
+        "data": {
+          "username": "Email Address",
+          "password": "Password",
```

Comment:
You can use global keys from here - https://github.com/home-assistant/core/blob/dev/homeassistant/strings.json